### PR TITLE
platform.hashing: add version 0.5.5, support conan v2

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **platform.hashing/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/platform.hashing/all/test_package/conanfile.py [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
 
+    def generate(self):
+        tc = CMakeToolchain(self)
+        if not is_msvc(self):
+            tc.variables["EXTRA_FLAGS"] = self._extra_flags
+            print(f"aaaaaa {self._extra_flags}")
```
Comment (jwillikers) [inline recipes/platform.hashing/all/test_package/conanfile.py:26]: Is this some debug output here?

**Review by jwillikers — CHANGES_REQUESTED** [review]

File: recipes/platform.hashing/all/test_package/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/platform.hashing/all/test_package/conanfile.py:26]: Oh, it's my mistake!
I will fix soon.
Thanks!

**Review by jwillikers — APPROVED** [review]

File: recipes/platform.hashing/all/test_package/conanfile.py (hunk 1 above)
Comment (jwillikers) [reply recipes/platform.hashing/all/test_package/conanfile.py]: No problem, LGTM!

**Review by uilianries — APPROVED** [review]
LGTM. Thank you for reporting it to the upstream and showing all those details!

**Review by jwillikers — APPROVED** [review]

**Pull request merged**